overlord/devicestate/devicemgr.go

```diff
@@ -479,6 +482,47 @@ func (m *DeviceManager) ensureBootOk() error {
 	return nil
 }
 
+func (m *DeviceManager) ensureInstalled() error {
+	m.state.Lock()
+	defer m.state.Unlock()
+
+	if release.OnClassic {
+		return nil
+	}
+
+	if m.ensureInstalledRan {
+		return nil
+	}
+
+	if m.operatingMode != "install" {
+		return nil
+	}
+
+	var seeded bool
+	err := m.state.Get("seeded", &seeded)
+	if err != nil {
+		return err
+	}
+	if !seeded {
+		return nil
+	}
+
+	if m.changeInFlight("install-system") {
+		return nil
+	}
+
+	m.ensureInstalledRan = true
+
+	tasks := []*state.Task{}
+	makeRunSystem := m.state.NewTask("make-run-system", i18n.G("Prepare system for run mode"))
```

Comment:
maybe s/make/setup/  and s/Prepare/Setup/

Reply:
It sounds appropriate, renaming.